lib/screens/play_room/play_room.dart

```diff
@@ -10,14 +10,32 @@ import 'life_stages.dart';
 import 'play_view.dart';
 import 'player_action.dart';
 
-class PlayRoom extends StatelessWidget {
+class PlayRoom extends StatefulWidget {
   const PlayRoom({Key key}) : super(key: key);
-  Size get _desktop => const Size(1440, 1024); // FIXME: 最終的には、左側のビューと右側のビューの最小サイズをトリガとして切り替える
+
+  @override
+  PlayRoomState createState() => PlayRoomState();
+}
+
+class PlayRoomState extends State<PlayRoom> {
+  bool isDisplayedResult;
```

Comment:
```suggestion
  bool isDisplayedResult = false;
```
とすれば initState の代入要らなそう